tests/drivers/retained_mem/api/boards/nrf54h20dk_nrf54h20_cpurad.overlay

```diff
@@ -0,0 +1,17 @@
+/ {
+	cpurad_tcm@23000000 {
+		compatible = "zephyr,memory-region", "mmio-sram";
+		reg = <0x23000000 DT_SIZE_K(32)>;
```

Comment:
This seems to redefine https://github.com/zephyrproject-rtos/zephyr/blob/e1cacb3a183968598e0baa12f3d7a84de3fe401a/dts/common/nordic/nrf54h20.dtsi#L242-L244. Is it intentional?

Then, `retainedmem` below seems to conflict with https://github.com/zephyrproject-rtos/zephyr/blob/e1cacb3a183968598e0baa12f3d7a84de3fe401a/boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpurad.dts#L25

Reply:
No, I think it a bug, I wasn't aware app & rad ram0's are utilized in devicetree. Should be fixed now